sdk/src/calibration_96tof1.cpp

```diff
@@ -97,8 +108,129 @@ aditof::Status Calibration::displayCalMap() const {
     return Status::OK;
 }
 
-aditof::Status Calibration::getAfeFirmware(const std::string &mode,
-                                           std::vector<uint16_t> &data) const {
+//! SaveCalMap - Save the entire calibration map
+/*!
+SaveCalMap - Saves the entire calibration map as binary to a file.
+\param calibration_map - Calibarion map to be displayed
```

Comment:
doc for parameters does not match the actual parameters

Reply:
Done